treeherder/log_parser/tasks.py

```diff
@@ -20,6 +19,8 @@
 logger = logging.getLogger(__name__)
 
 
+# XXX guessing I'm removing this decorator, then adding this decorator to the
+# parser tasks?
```

Comment:
Yeah, exactly.  So add it to `store_failure_lines` and the other one.  I didn't think this through 100%, so some massaging may be necessary there.